src/navigation/NavBar.js

```diff
@@ -26,77 +25,66 @@ const NavLink = ({
   const href = `#${to}`
   const active = exact ? hash === href : hash && hash.startsWith(href)
   const anchorClass = classnames({
-    'bg-white-10': active,
+    'bg-white-10 navbar-item-active': active,
     'o-50 no-pointer-events': disabled
-  }, ['dib db-l pv3 white no-underline f5 hover-bg-white-10 tc'])
+  }, ['dib db-l pt2 pb3 pv3-l white no-underline f5 hover-bg-white-10 tc bb bw2 bw0-l b--navy'])
   const svgClass = classnames({
     'o-100': active,
     'o-50': !active
   }, ['fill-current-color'])
 
   return (
     // eslint-disable-next-line jsx-a11y/anchor-is-valid
-    <a href={disabled ? null : href} className={anchorClass} style={{ borderLeft: active ? '5px solid rgba(201, 210, 215, .4)' : '' }} role='menuitem' title={children}>
-      <span className={`dib ${open ? 'dt-l' : ''}`}>
-        <span className={`dib dtc-l v-mid ${open ? 'pl3 pl5-l' : 'ph3'}`} style={{ width: 50 }}>
-          <Svg width='50' className={svgClass} />
-        </span>
-        <span className={`${open ? 'dib dtc-l' : 'dn'} pl2 pl3-l pr3 tl-l v-mid `}>
+    <a href={disabled ? null : href} className={anchorClass} role='menuitem' title={children}>
+      <div className='db ph2 pv1'>
+        <div className='db'>
+          <Svg width='50' role='presentation' className={svgClass} />
+        </div>
+        <div className={`${active ? 'o-100' : 'o-50'} db f6 tc montserrat ttu fw1 `} style={{ whiteSpace: 'pre-wrap' }}>
           {children}
-        </span>
-      </span>
+        </div>
+      </div>
     </a>
   )
 }
 
-export const NavBar = ({ t, open, onToggle }) => {
+export const NavBar = ({ t }) => {
   const codeUrl = 'https://github.com/ipfs-shipyard/ipfs-webui'
   const bugsUrl = `${codeUrl}/issues`
   const gitRevision = process.env.REACT_APP_GIT_REV
   const revisionUrl = `${codeUrl}/commit/${gitRevision}`
   return (
     <div className='h-100 fixed-l flex flex-column justify-between' style={{ overflowY: 'auto', width: 'inherit' }}>
       <div className='flex flex-column'>
-        <button className='pointer navy pv3 pv4-l' onClick={onToggle} aria-label={t('collapseNavbar')}>
-          <img className='center' style={{ height: 70, display: open ? 'block' : 'none' }} src={ipfsLogoText} alt='IPFS' title='Toggle navbar' />
-          <img className='center' style={{ height: 70, display: open ? 'none' : 'block' }} src={ipfsLogo} alt='IPFS' title='Toggle navbar' />
-        </button>
+        <a href="#/welcome" role='menuitem' title={t('welcome:description')}>
+          <div className='pt3 pb1 pv4-l'>
+            <img className='center db-l dn' style={{ height: 100 }} src={ipfsLogoTextVert} alt='IPFS' role='presentation' />
```

Comment:
You shouldn't add both alt and `role="presentation"` here. It's counter-intuitive.

You can simply do `alt=''` if you want screen readers to skip this image 😄

Reply:
My bad, thanks for finding 😊